README.md

```diff
@@ -49,6 +49,18 @@ Some extensions may specify more specific rules for this field.
 
 This field superceedes the `sar:product_type` field.
 
+#### product:acquisition_type
+
+The product acquisition type in this extension is an enumeration text that providers can use to descibe their product types.
+As described in the [specifications](https://docs.ogc.org/is/10-157r4/10-157r4.html):
```

Comment:
The link is very unspecific. I can't find related information for this field easily.

Reply:
Linked documentation doesn't have relative links, see chapter _8.1.1 EarthObservationMetaData_, Table 5 describes `acquisitionType` field - our reflection is 1:1 to the spec

Reply:
Okay, then mention this in the README please so that readers can find it more easily.

Reply:
Fixed in https://github.com/stac-extensions/product/pull/5/commits/e95b325f8decb50c25809b4a7a9373b4ed6ca765

Reply:
Maybe I'm blind, but I don't see any mention of the specific table and property to look for?!

Reply:
@m-mohr description enhanced in README in https://github.com/stac-extensions/product/pull/5/commits/a4128695ea95a9a1015cdc4d5aa8a905baf103a9